From 59a2e4940917bdf1ad1a5217d0d0c216e6801178 Mon Sep 17 00:00:00 2001
From: HungTran <hung.tran.jy@rvc.renesas.com>
Date: Fri, 30 Oct 2015 18:41:54 +0700
Subject: [PATCH] Doorphone: Change to use waylandwrlsink instead of
 waylandsink

In new environment, there are 4 waylandsink version
(probably due to conflict source code from many source)

They are waylandsink, waylandwrlsink, waylandfssink and waylandmiraclesink
The version for Doorphone is waylandwrlsink now.

Signed-off-by: HungTran <hung.tran.jy@rvc.renesas.com>
---
 src/plugins/gstreamer/H264Streaming/qgstreamerH264Streamingsession.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/gstreamer/H264Streaming/qgstreamerH264Streamingsession.cpp b/src/plugins/gstreamer/H264Streaming/qgstreamerH264Streamingsession.cpp
index cd12864..00c99e2 100755
--- a/src/plugins/gstreamer/H264Streaming/qgstreamerH264Streamingsession.cpp
+++ b/src/plugins/gstreamer/H264Streaming/qgstreamerH264Streamingsession.cpp
@@ -266,7 +266,7 @@ QGstreamerH264StreamingSession::QGstreamerH264StreamingSession(QObject *parent)
     filter1 = gst_element_factory_make ("vspmfilter","vspm-filter1");
     encoder = gst_element_factory_make ("omxh264enc","omx-encoder");
     decoder = gst_element_factory_make ("omxh264dec","omx-decoder");
-    videosink = gst_element_factory_make ("waylandsink","video-output");
+    videosink = gst_element_factory_make ("waylandwrlsink","video-output");
     capfilter_vspm = gst_element_factory_make("capsfilter", "caps-filter");
     capfilter_vspm1 = gst_element_factory_make("capsfilter", "caps-filter1");
     videoIdentity_parse = gst_element_factory_make("capsfilter", NULL);
-- 
1.9.1

